src/prototypes/articles/example/example.twig

```diff
@@ -0,0 +1,42 @@
+<div id="prototype-articles-example">
+  <div class="articles">
+    {% for article in feed.items %}
+      <article>
+        <header>
+          <h2>
+            <a href="{{article.url}}">
+              {{article.title}}
+            </a>
+          </h2>
+        </header>
+        <p>
```

Comment:
🤔 Would it be appropriate to wrap content after `<header>` and before `<footer>` in a `<section>` tag?

Reply:
No, `<section>` would not be appropriate there.

Reply:
Gotcha, I saw [this](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article#Examples) example and wasn't sure.

Reply:
That example confuses me.

The good news is that this is just an example: For prototyping purposes, you can "save as" and write whatever markup you want! 🎉

Reply:
😄 Thank you sir.